src/client/pages/ConnectPayment/sections/ConnectPayment.tsx

```diff
@@ -5,105 +5,131 @@ import { Market, useMarket } from 'components/utils/CurrentLocale'
+    padding-right: 1rem;
+    padding-top: 6.75rem;
+    flex-direction: column-reverse;
+    width: 100%;
+  }
+`
 
-const TextColumn = styled('div')({
-  width: '60%',
-  paddingRight: 20,
-  paddingLeft: 20,
-  paddingBottom: 40,
+const TextColumn = styled.div`
+  width: 60%;
+  padding-bottom: 2.5rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+  }
+`
 
-  [`@media (max-width: ${BP}px)`]: {
-    width: '100%',
-    textAlign: 'center',
+const ImageColumn = styled.div`
+  width: 40%;
+  margin-top: 0.5rem;
+  padding-left: 2rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+    padding: 0;
+    text-align: center;
   },
-})
-
-const ImageColumn = styled('div')({
-  width: '40%',
-  paddingRight: 20,
-  paddingLeft: 20,
-  paddingBottom: 40,
+}
+`
 
-  [`@media (max-width: ${BP}px)`]: {
-    width: '100%',
-    textAlign: 'center',
-  },
-})
+const Header = styled.h1`
+  margin-top: 0;
+  margin-bottom: 2.25rem;
+  font-size: 3rem;
+  color: ${colorsV3.white};
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    text-align: center;
+    margin-bottom: 2rem;
+    font-size: 2rem;
+    line-height: 2rem;
+  }
+`
 
-const Header = styled('h1')({
-  marginTop: '0px',
-  marginBottom: '30px',
-  fontSize: '56px',
-  lineHeight: '60px',
-  color: colorsV3.white,
-  [`@media (max-width: ${MOBILE}px)`]: {
-    fontSize: '44px',
-    lineHeight: '52px',
-  },
-})
+type HeaderPartProps = {
+  addWhiteSpace?: boolean
+}
+const HeaderPart = styled.span<HeaderPartProps>`
+  display: block;
+  font-family: ${fonts.FAVORIT};
+  font-weight: 400;
 
-const HeaderPart = styled('span')({
-  display: 'block',
-  fontFamily: fonts.FAVORIT,
-  fontWeight: 400,
-})
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    display: initial;
+    ${(props) =>
+      props.addWhiteSpace &&
+      `
+      :before {
+        content:'\\00a0';
+      }`}
+`
+const ConnectText = styled.div`
+  width: 65%;
+  margin-bottom: 1rem;
+  font-size: 1.25rem;
+  color: ${colorsV3.white};
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+  }
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    font-size: 1rem;
+  }
+`
+const InstructionWrapper = styled(ConnectText)`
+  margin-bottom: 1rem;
+  color: ${colorsV3.gray500};
+`
 
-const ConnectText = styled('div')({
-  width: '65%',
-  marginBottom: '45px',
-  [`@media (max-width: ${BP}px)`]: {
-    width: '100%',
-  },
-})
+const ConnectPaymentImage = styled.img`
+  width: 80%;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 40%;
+    margin-bottom: 2rem;
+  }
+`
 
-const ConnectPaymentImage = styled('img')({
-  marginBottom: '30px',
-  marginTop: '30px',
-  marginLeft: 'auto',
-  width: '100%',
-  [`@media (max-width: ${BP}px)`]: {
-    marginRight: 'auto',
-    width: '80%',
-  },
-  [`@media (max-width: ${MOBILE}px)`]: {
-    width: '100%',
-  },
-})
+const InstructionsList = styled.ol`
+  list-style: none;
+  padding: 0;
+  counter-reset: styled-number;
+`
+const InstructionItem = styled.li`
+  counter-increment: styled-number;
+  :: before {
```

Comment:
```suggestion
  :before {
```

Reply:
For some context: https://css-tricks.com/to-double-colon-or-not-do-double-colon/
TL;DR double colon is the correct syntax for pseudo-elements, but the old syntax that I'm used to (single colon) works for both pseudo-elements and pseudo-selectors and hence you don't have to think about what is what 😄

Reply:
Ah and the extra space as you pointed out @styrelius , didn't see that at first 👍